mllib-dal/src/main/java/com/intel/oap/mllib/LibLoader.java

```diff
@@ -62,6 +62,7 @@ private static synchronized void loadLibMLlibDAL() throws IOException {
     // oneDAL Java API doesn't load correct libtbb version
     // See https://github.com/oneapi-src/oneDAL/issues/1254
     // Workaround: Load packaged libtbb & libtbbmalloc & libJavaAPI.so manually
+    System.loadLibrary("onedal_thread");
```

Comment:
I am not sure we should add this in the code. Do you have a reproducer for the issue?